src/commands.rs

```diff
@@ -919,24 +919,25 @@ struct GitCloneArgs {
 /// Push to a Git remote
 ///
 /// By default, pushes any branches pointing to `@`, or `@-` if no branches
-/// point to `@`. Use `--branch` to push a specific branch. Use `--all` to push
-/// all branches. Use `--change` to generate a branch name based on a specific
-/// commit's change ID.
+/// point to `@`. Use `--branch` to push specific branches. Use `--all` to push
+/// all branches. Use `--change` to generate branch names based on the change
+/// IDs of specific commits.
 #[derive(clap::Args, Clone, Debug)]
 #[command(group(ArgGroup::new("what").args(&["branch", "all", "change"])))]
```

Comment:
I suppose we could make `jj git push --branch my-branch --change abc123` allowed. But that can come later (or in this PR if you want)

Reply:
This is mildly awkward to implement, mainly for the silly reason that I'm undecided what the transaction names should be.

Reply:
Those strings are just hints that are supposed to help the user figure out what they were doing. When using the CLI (which is of course our only UI so far), the full arguments are most useful to me anyway. I'd be fine if the description was just "Push branches to Git remote 'origin'"/"Push changes to Git remote 'origin'"/"Push branches and changes to Git remote 'origin'"  or something like that. I haven't put much thought into the descriptions myself so far.

Reply:
Added a TODO